settings.gradle.kts

```diff
@@ -10,6 +18,12 @@ pluginManagement {
 		id("org.asciidoctor.convert") version "1.5.8.1"
 		id("me.champeau.gradle.jmh") version "0.4.8"
 		id("io.spring.nohttp") version "0.0.3.RELEASE"
+		id("biz.aQute.bnd.builder")
+	}
+
+	repositories {
+		gradlePluginPortal()
+		mavenCentral()
```

Comment:
Why do we need `mavenCentral()`?

Reply:
Also due to the issue with the gradle plugin portal...